bin/autoUpdateRepo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+#:Title: autoUpdateRepo.sh
+#:Author: Alan Coleman
+
+#changes directory to folder containing repositories
+cd ~/ELK_SRC_HOME
```

Comment:
We should not assume ELK_SRC_HOME will be located in your home directory.

We should assume the user will set up an environments variable, such as:
`export ELK_POC_HOME=~/elk-poc`

Such instructions would be in the README.

Reply:
currently the script does not work from crontab when using environmental variables, fixing this has been added as a future task to asana